README.md

```diff
@@ -208,102 +273,155 @@ supported. For example, we can build a layer with a delay differential equation
 like:
 
 ```julia
-function delay_lotka_volterra(du,u,h,p,t)
+# Define the same LV equation, but including a delay parameter
+function delay_lotka_volterra!(du, u, h, p, t)
   x, y = u
   α, β, δ, γ = p
-  du[1] = dx = (α - β*y)*h(p,t-0.1)[1]
-  du[2] = dy = (δ*x - γ)*y
+  du[1] = dx = (α   - β*y) * h(p, t-0.1)[1]
+  du[2] = dy = (δ*x - γ)   * y
 end
-h(p,t) = ones(eltype(p),2)
-u0 = [1.0,1.0]
-prob = DDEProblem(delay_lotka_volterra,u0,h,(0.0,10.0),constant_lags=[0.1])
 
+# Initial parameters
 p = [2.2, 1.0, 2.0, 0.4]
+
+# Define a vector containing delays for each variable (although only the first
+# one is used)
+h(p, t) = ones(eltype(p), 2)
+
+# Initial conditions
+u0 = [1.0, 1.0]
+
+# Define the problem as a delay differential equation
+prob_dde = DDEProblem(delay_lotka_volterra!, u0, h, (0.0, 10.0), constant_lags = [0.1])
+
 function predict_dde(p)
-  Array(concrete_solve(prob,MethodOfSteps(Tsit5()),u0,p,saveat=0.1,sensealg=TrackerAdjoint())
+  return Array(concrete_solve(prob_dde, MethodOfSteps(Tsit5()),
+                              u0, p, saveat = 0.1,
+                              sensealg = TrackerAdjoint()))
 end
-loss_dde(p) = sum(abs2,x-1 for x in predict_dde(p))
+
+loss_dde(p) = sum(abs2, x-1 for x in predict_dde(p))
 loss_dde(p)
 ```
 
-Notice that we chose `sensealg=TrackerAdjoint()` to utilize the Tracker.jl
+```
+369.9247935840374
```

Comment:
I am not sure adding the output values manually is a good idea

Reply:
I left them in to give the reader a sense of what to expect when executing. 

As a dynamic content document format, Github markdown is not good enough. I thought about writing something in Julia markdown to generate this file. Maybe that would be a better solution.